setup.cfg

```diff
@@ -52,6 +52,6 @@ combine_as_imports=True
 line_length=88
 # isort can't be applied to yt/__init__.py because it creates circular imports
 skip = venv, doc, benchmarks, yt/__init__.py, yt/extern
-known_third_party = IPython, nose, numpy, sympy, matplotlib, unyt, git, yaml, dateutil, requests, coverage, pytest, pyx
+known_third_party = IPython, nose, numpy, sympy, matplotlib, unyt, git, yaml, dateutil, requests, coverage, pytest, pyx, glue
```

Comment:
is this glue addition relevant to this PR?

Reply:
Isort side effect. Maybe with the updated version it's not necessary anymore but I had to add it otherwise the isort hook was unhappy about a line that's actually fine.

Reply:
I guess I'm confused why isort would want another package added to this line? these other third party packages are hard dependencies or used for development. glue should be one of our optional dependencies.

Reply:
There's no différenciation between hard and optional deps as far as isort is concerned. It just means "if you see THIS being imported, treat it as third party" and indeed the one line where that's important is not a top level import but buried somewhere in a function. Isort only recently gained the ability to sort even those so it's not too surprising that we're seeing évolutions there :)